paasta_tools/tron_tools.py

```diff
@@ -205,24 +205,40 @@ def get_cmd(self):
             command = "unset MESOS_DIRECTORY MESOS_SANDBOX; " + command
         return command
 
+    def get_vault_token_file(self):
+        return self.config_dict.get(
+            "vault_token_file", "/var/spool/.paasta_vault_token"
```

Comment:
nit: Can we make `/var/spool/.paasta_vault_token` a global constant?

Reply:
yeah or maybe put it in the SystemPaastaConfig?